src/main/java/evolvio/render/IRenderer.java

```diff
@@ -0,0 +1,7 @@
+package evolvio.render;
+
+import java.awt.Graphics2D;
+
+public interface IRenderer<T> {
```

Comment:
Why use IRender instead of Renderer? It is clear that this represents an interface isn't it (and most modern IDE use different Icons for interfaces, classes and abstract classes too). Pls get rid of the I.

Reply:
Probably more of a personal preference, I slightly prefer it but they're fair points